xml/decoderIndex.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I did a "git checkout origin/master --  xml/decoderIndex.xml" which I suspect solves this one.

Reply:
Question: Do we need entries in this for our new decoders, or is it enough to have them in xml/decoders?

Reply:
Running `ant remakedecoderindex` updates this file from the individual decoder files. You're welcome to run that. We also do it every time as we build releases.  Your models will be present in it either way.

Reply:
Thinking about it a bit more, might be better to let us do it as part of the release later.  That'll reduce the odds of a merge conflict.

Reply:
> Thinking about it a bit more, might be better to let us do it as part of the release later. That'll reduce the odds of a merge conflict.

That's why I prefer to avoid adding decoderIndex.xml to a PR with other changes. If I do it, I make it the sole file in a new PR after the others have been merged.

Reply:
Understood, thanks for the details